flow/testing/diff_context_test.cc

```diff
@@ -0,0 +1,49 @@
+#include "diff_context_test.h"
```

Comment:
for these files i think we can include them in the `BUILD.gn` file iff `enable_partial_repaint` or some equivalent is passed from `flutter/tools/gn` script.

Reply:
I could create an argument for it (that you can change in args.gn).